help/en/Acknowledgements.shtml

```diff
@@ -239,6 +239,10 @@
         "http://www.quaker-valley.com/CTC/JMRI_Panel_Tutorial.html">
         online panel tutorial</a>.</li>
 
+        <li>Geoff Bunzxa, <!-- gbglacier -->
```

Comment:
I _think_ this is a typo - shouldn't it be `Geoff Bunza`?

Reply:
Thanks - have fixed.